sdk/ai/azure-ai-projects/azure/ai/projects/_types.py

```diff
@@ -10,7 +10,6 @@
 
 if TYPE_CHECKING:
     from . import models as _models
-    from .. import models as _models
```

Comment:
This file will be generated by TSP. Mypy gives the same error.

Reply:
@nick863 you added this line in your previous PR, which redefines _models. Also ".. import models" will not work since it's not in the upper folder, it's in the same folder as _types.py. I'm assuming that was an error, therefore I'm removing it.